examples/kotlin-smartapp/src/main/kotlin/routes/SmartApp.kt

```diff
@@ -11,11 +13,14 @@ import com.smartthings.sdk.client.ApiClient
 import com.smartthings.sdk.client.methods.DevicesApi
 import com.smartthings.sdk.smartapp.core.Response
 import com.smartthings.sdk.smartapp.core.SmartApp
+import com.smartthings.sdk.smartapp.core.models.AppLifecycle
+import com.smartthings.sdk.smartapp.core.models.ExecutionRequest
 import com.smartthings.sdk.smartapp.core.models.EventResponseData
 import com.smartthings.sdk.smartapp.core.models.InstallResponseData
 import com.smartthings.sdk.smartapp.core.models.UninstallResponseData
 
-val api = ApiClient()
+val api = ApiClient().setBasePath("https://api.smartthings.com/v1/")
```

Comment:
If we are using prod, do we actually need to set the base path? Thought it defaulted to it.

Reply:
You are correct. I saw the 401 because of a problem with the api request inside install and when I ran again with updated code, it called update because I hadn't uninstalled the app. I've made both of them use the same code now.